demo/oauth2-reddit.R

```diff
@@ -15,3 +15,11 @@ token <- oauth2.0_token(reddit, app,
   scope = c("read", "modposts"),
   use_basic_auth = TRUE
 )
+
+# 3b. If get 429 too many requests, the default user_agent is overloaded.
+# If you have an application on Reddit then you can pass that using:
+token <- oauth2.0_token(reddit, app,
```

Comment:
Please only use two spaces to indent - http://style.tidyverse.org/syntax.html#long-lines